contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -339,14 +339,13 @@ class WaitForConnectionToTargetInstanceAndProcess(E2ETestCase):
     """
```

Comment:
You should update this comment and the one at the top of `orbit_connect_to_target.py` as you are no longer checking the `ConnectToTargetDialog`.

Reply:
Oh right, I missed the comments. Fixed, thanks.